Build fails due to git invocation fail

# Summary
Build fails due to git invocation fail

# Steps to reproduce
- Platform: Desktop, WIndows 11
- Framework Version: .NET Core 7
- Download release Silk.NET-2.17.1
1. Run `build.cmd`
2. Get error 
`...\Silk.NET-2.17.1\build\props\common.props(36,9): error MSB3073: The command "git show -s "--format=%%cI"" exited with code 128.`

** I think it's related to the `format` parameter passed to git command **

--- a/build/props/common.props
+++ b/build/props/common.props
@@ -26,7 +26,7 @@
         </Description>
     </PropertyGroup>
     <!-- Date/time based versioning (mainly used by natives) -->
-    <Target Name="SilkAddDateTime" Condition="'$(SilkGitDirForVersion)' != ''">
+    <Target Name="SilkAddDateTime" Condition="'$(SilkGitDirForVersion)' != '' and Exists('$(SilkGitDirForVersion)/.git')">
         <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
             <SilkGitDateTimeFormat>%25%25cI</SilkGitDateTimeFormat>
         </PropertyGroup>

